esp/templates/program/modules/autoschedulerfrontendmodule/autoscheduler.html

```diff
@@ -0,0 +1,134 @@
+{% extends "main.html" %}
+
+{% block title %}Run the Automatic Scheduling Assistant{% endblock %}
+
+{% block subsection_name %}Autoscheduler Options{% endblock %}
+
+{% block content %}
+
+<script>
+var program_url_base = '{{ program.getUrlBase }}';
+</script>
+<script type="text/javascript" src="/media/scripts/program/modules/autoschedulerfrontend.js"></script>
+
+<h1>Run the Automatic Scheduling Assistant</h1>
+
+<p>A list of options is below. You should generally be able to leave them at
+their defaults, but you can also play around with them if you prefer. If you
+think the defaults should be changed, you can create or modify the appropriate
+Tag; see the bottom of the page for more information. When you're ready, run
+the scheduling assistant, then save the results if you are satisfied.</p>
+
+<form id="autoschedulerform" action="javascript:runAutoscheduler()" class="form-inline" method="POST">
+<center>
+    <h2>Search Options</h2>
+<table width="100%" cellpadding="5">
+{% for option_key, option_value in search.items %}
+<tr>
+	<th>{{ option_key }}</th>
+	<td><input type="text" class="input-small" name="autoscheduler_search_{{ option_key }}" value="{{ option_value.0 }}" /></td>
+	<td>{{ option_value.1 }}</td>
+</tr>
+{% endfor %}
+</table>
+    <h2>Constraint Options
+    	<a href="javascript:toggleShowOptions('constraints')">
```

Comment:
Is there a reason you have to wrap the buttons in a tags? An onclick handler should work fine, right?

```
<button type="button" onclick="toggleShowOptions(...)">Show/hide...</button>
```